database/migrations/2020_05_05_060000_create_cart_items_table.php

```diff
@@ -35,7 +35,7 @@ public function up()
             $table->unsignedInteger('tax')->default(0);
 
             $table->dateTime('expires_at');
-            $table->unsignedInteger('location_id')->nullable();
+            $table->unsignedBigInteger('location_id')->nullable();
```

Comment:
The non-FK addition of  `location_id` fields looks like it was done w/the wrong field size.  Because its not an FK, mysql won't complain, but something that should be changed to `BigInteger`